extensions/binding/org.eclipse.smarthome.binding.astro/README.md

```diff
@@ -171,13 +171,13 @@ astro:sun:home [ geolocation="52.5200066,13.4049540,100", interval=60 ] {
 Items:
 
 ```
-DateTime Sunrise_Time       "Sunrise [%1$tH:%1$tM]"  { channel="astro:sun:home:rise#start" }
-DateTime Sunset_Time        "Sunset [%1$tH:%1$tM]"   { channel="astro:sun:home:set#start" }
-Number   Azimuth            "Azimuth"                { channel="astro:sun:home:position#azimuth" }
-Number   Elevation          "Elevation"              { channel="astro:sun:home:position#elevation" }
-String   MoonPhase          "MoonPhase"              { channel="astro:moon:home:phase#name" }
-Number   Total_Radiation    "Radiation"              { channel="astro:sun:home:radiation#total" }
-Number   Diffuse_Radiation  "Diffuse Radiation"      { channel="astro:sun:home:radiation#diffuse" }
+DateTime Sunrise_Time       "Sunrise [%1$tH:%1$tM]"                   { channel="astro:sun:home:rise#start" }
+DateTime Sunset_Time        "Sunset [%1$tH:%1$tM]"                    { channel="astro:sun:home:set#start" }
+Number   Azimuth            "Azimuth"                                 { channel="astro:sun:home:position#azimuth" }
+Number   Elevation          "Elevation"                               { channel="astro:sun:home:position#elevation" }
+String   MoonPhase          "MoonPhase"                               { channel="astro:moon:home:phase#name" }
+Number   Total_Radiation    "Radiation [%.2f W/m\u00B2]"              { channel="astro:sun:home:radiation#total" }
```

Comment:
Does anyone know whether the transformation supports UTF-8 characters? If it does this PR looks good for me.

Reply:
@emme-g Do you know? Has it been tested?

Reply:
Hi Markus,
the only modification I did was to add in the documentation the specification about the dimensions W/m2 for the radiation channels. If you ask if the "\u00B2" to show the character "2" as superscript has been tested, I've seen it in the documentation of another Openhab binding (Air Quality https://github.com/openhab/openhab2-addons/tree/master/addons/binding/org.openhab.binding.airquality).
In my Openhab system now I use these subscripts and superscripts, and in my sitemap they are showed right in the way I expect both in the Air Quality items and in the Astro radiation items.
I just noticed that my modification erased the line about the diffuse radiation channel, maybe it's not a problem because it was just an example but I could re-add it if in your opinion it would be better.

Reply:
Yes, this works